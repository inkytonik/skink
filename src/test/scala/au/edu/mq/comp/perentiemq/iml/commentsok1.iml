// Hello there

def commentsok1(a) : (b) {

    a /* Foo */ = 1;
    a = 1; // Foo

    /*
     * A non-nested comment.
     */

    /* A nested comment
      /* Woo!! */
      Made it
      */

}
