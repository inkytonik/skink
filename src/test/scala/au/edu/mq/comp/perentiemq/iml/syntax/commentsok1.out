Program (
    List (
        FunDef (
            "commentsok1",
            List (Arg ("a")),
            List (Arg ("b")),
            Block (
                List (
                    Assign (List (Var ("a")), List (IntLit (1))),
                    Assign (List (Var ("a")), List (IntLit (1))))))))
def commentsok1(a): (b) {
    a = 1;
    a = 1;
}

